### ISSUE TITLE
Fail mvn package for maven-surefire-plugin

### ISSUE THREAD

**Issue by hidcontrol** [camunda/camunda-bpm-reactor#100]
solve: change vesion in pom.xml 2.20.1->2.19.1

**Comment by GabrielCasaro** [comment]
As stated above, it does work changing version. However, downgrading versions should be discouraged.
As [informed here](https://bugzilla.redhat.com/show_bug.cgi?id=1572708), 

> Running any tests on surefire 2.20 with JDK 10 results in NPE

 and 

> Note this is not an issue in maven surefire plugin 2.21 as it's been fixed there
 
Updating maven-surefire-plugin version to 2.21.0 works like a charm.

```
<plugin>
     <artifactId>maven-surefire-plugin</artifactId>
     <version>2.21.0</version>
     <configuration>
          <runOrder>${surefire.runOrder}</runOrder>
     </configuration>
</plugin>
```

**Comment by jangalinski** [comment]
Cool, thanks ... Could you put this in a PR? I would only find it fair to give you the commit instead of c&p this from your comment ...

**Comment by jangalinski** [comment]
fixed with #111
